packages/ruby/install-ruby.ts

```diff
@@ -61,18 +61,27 @@ function getRubyPath(meta: Meta, gemfileContents: string) {
 // process.env.GEM_HOME), and returns
 // the absolute path to it
 export async function installBundler(meta: Meta, gemfileContents: string) {
+  const { gemHome, rubyPath, gemPath, vendorPath, runtime } = getRubyPath(
+    meta,
+    gemfileContents
+  );
+
   // If the new File System API is used (`avoidTopLevelInstall`), the Install Command
   // will have already installed the dependencies, so we don't need to do it again.
   if (meta.avoidTopLevelInstall) {
```

Comment:
Don't we need to check if it's the correct path? Did I oversee that in  the previous PR?

Reply:
No, because this function installs `bundler` (which is used for `bundle install`) and that CLI is already available ahead of time, because it's used for the `Gemfile` installation in the File System API.

What might be is that that `bundler` is located elsewhere and this function has to be adjusted accordingly, but I don't think so, and I'll verify that in the Deployment once it's published.

What might also be is that our `Gemfile` installation in the File System API doesn't support installing custom versions of `bundler`, which I think this function does, but I'll verify that too.